src/routes/Categories.jsx

```diff
@@ -0,0 +1,10 @@
+export default function Categories() {
```

Comment:
- [ ] Nice working on your categories page, as a requirement this page is supposed to only dispaly a `button` with text "Check status".

Reply:
oh, Sorry I did not see that.
It has been fixed.
![image](https://user-images.githubusercontent.com/13853450/178085509-46eaa177-ac67-4aef-9cac-002d2163bbe4.png)